README.md

```diff
@@ -82,6 +82,18 @@ test-suite doctests:
 Notes
 -----
 
+* `Cabal-2.0` can be finicky about whether it builds the library component or
```

Comment:
I'm not sure about the wording. `Cabal-2.0` is IMHO *smart* to build components independently, if there are no explicit inter-dependencies!

And adding the dependency is kind-of hack too, which just works *better* and with less hassle in practice (especially in multi-package projects, where `doctests` without dependencies would be definitely built first).

Pity we didn't notice that earlier, as e.g. `multi-ghc-travis` scripts do `new-build --enable-tests` first, and only after that do `new-test`.